tests/utils/assertions.ts

```diff
@@ -11,5 +13,8 @@ export async function assertNumberOfModalityLoadBadges({
   page: Page;
   expectedCount: number;
 }) {
-  await expect(page.locator('css=div[data-cy^="ModalityLoadBadge-"]')).toHaveCount(expectedCount);
+  const overlayPageObject = new OverlayPageObject(page);
```

Comment:
Shouldn't this page object be one of those that get created in fixture.ts?

Reply:
Yes, but the utility does not have access to it directly. And I preferred instantiating the class by using the page object it already received, than receiving more parameters in the function. I think that for the user of the utility it looks cleaner. What do you think?